Blockly/js/update.js

```diff
@@ -2026,3 +2026,15 @@ function CreateAxmlfile(){
 		 }
 	}
 }
+function inputfileselect(){
+	document.getElementById('file-selector').click();
+}
+
+async function importFile(event){
+const fileList = event.target.files;

```

Comment:
@mahdi943 I would like to fix the indentation here.
```suggestion
    const fileList = event.target.files;
```